content/docs/about/use-cases.md

```diff
@@ -0,0 +1,30 @@
+
+## Experimentation with training an ML model
+
+Rapid experimentation is critical to building high quality machine learning models quickly. Kubeflow offers a user-friendly UI interface that allows you to track and compare experiments. You can decide later on which experiment was the best and use it as a main source for your future steps. On top of that Kubeflow 1.0 provides stable software sub-systems for model training including Jupyter notebooks, Fairing, popular ML training operators such as Tensorflow and Pytorch that run efficiently and securely in Kubernetes isolated namespaces. The ML training operators simplify configuration and operations of scaling ML training tasks. In addition, the Kubeflow has delivered Critical User Journeys(CUJs), such as the build, train and deploy, which provide end-to-end workflows that speed development. Please let more on the CUJs [here](http://bit.ly/kf_roadmap).
+
+## End to end Hybrid and Multi-clouds ML workloads
+
+The development of ML models can require hybrid and multi-cloud portability and secure sharing between teams, clusters and clouds. If you need to develop on your laptop, train with GPU on your on-prem cluster and serve in the cloud, Kubeflow provides the portability to support fast experimentation, rapid training and robust deployment in the same or different environments with minimal operational overhead.
+
+## Tuning the model hyperparameters during training
+
+During the model development part hyperparameters are often hard to tune. Tuning hyperparameters is critical for model performance and accuracy. Manually configuring hyperparameters is time consuming. Kubeflow’s hyperparameter tuner (Katib) provides an automated way to match your objectives. This automation can save days of model testing compute time (freeing up valuable infrastructure), and speed the delivery of improved models.
+
+## CI/CD for ML
+
+Kubeflow currently doesn’t have a dedicated tool for this purpose. But our users have been using the Pipelines component and it worked really well for them. Kubeflow Pipelines can be used to create reproducible workflows. These workflows automate the steps needed to build a ML workflow, which delivers consistency, saves iteration time, and helps in debugging, auditability and compliance requirements.
```

Comment:
At the end of this doc, it'd be good to add a "next steps" section and provide links to the main components that people can read to follow up on the content in this doc.

Something like this:

```
## Next steps

See these docs for more information on the topics covered above:

* [Hyperparameter tuning with Katib](/docs/components/hyperparameter-tuning/)
* And so on...
```